packages/teams-js/src/public/search.ts

```diff
@@ -1,8 +1,20 @@
-import { sendMessageToParent } from '../internal/communication';
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
You shouldn't need to disable linting rules for this change. Why was this necessary?

Reply:
I see what happened it was my mistake I used the search initialize instead of the app one, will correct it now.